.github/workflows/ci-format.yml

```
@@ -58,13 +58,18 @@ jobs:
           . ${{env.venv_dir}}/bin/activate
           pip install --upgrade pip wheel
           pip install .[camera-obs,dev,extras,opendrive,ros,test,train]
-      - name: Get changed files
+          # install the specific version we want for pytest, just in case...
+          pip install pytype==2022.1.13
+      - name: Get changed files on branch since branching
         id: changed-files
-        uses: tj-actions/changed-files@v13.2  
-        with:
-          base_sha: origin/develop     
+        shell: bash
+        run: |
+          cd $GITHUB_WORKSPACE
+          CHANGED=$(git diff --diff-filter="AM" --name-only --ignore-submodules=all origin/develop... | awk -v d=" " '{s=(NR==1?s:s d)$0}END{print s}')
+          echo "::set-output name=changed_files::$CHANGED"
       - name: Check types
-        if: contains(steps.changed-files.outputs.all_changed_files, '.py')
+        if: contains(steps.changed-files.outputs.changed_files, '.py')
         run: |
           . ${{env.venv_dir}}/bin/activate
-          pytype --disable=pyi-error ${{steps.changed-files.outputs.all_changed_files}}
\ No newline at end of file
+          pytype --disable=pyi-error ${{steps.changed-files.outputs.changed_files}}
```

```suggestion
          pytype -d pyi-error import-error ${{steps.changed-files.outputs.changed_files}}
```

With this change:
+ All third party imported libraries will be ignored.
+ Third party libraries do not need to be ignored individually in each file. 
+ Additionally, the time cosuming `pip install .[camera-obs,dev,extras,opendrive,ros,test,train]` command above can be changed to the simpler `pip install .[dev]` . Option `dev` was kept so that we use the same pytype version from setup.py file.

Not sure if there are any undesirable side-effects which arises from completely ignoring all import-errors.

---

Nice!  Thanks for the suggestion.  I'll make this change now.